pkg/server/api/bundle/v1/service_test.go

```diff
@@ -511,8 +511,11 @@ func TestAppendBundle(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			test.logHook.Reset()
 			test.setBundle(t, sb)
+			// Make sure no error is set to next call
```

Comment:
Maybe a better thing to do is to initialize a new `test` for each test case instead of trying to clean up/reset state on the existing test.

Reply:
cleaned